package.json

```diff
@@ -22,12 +22,17 @@
   "author": "Daniel G. Taylor <danielgtaylor@gmail.com>",
   "license": "MIT",
   "devDependencies": {
+    "chai": "^3.5.0",
     "coffee-coverage": "^1.0.1",
     "coffee-script": "^1.10.0",
     "coffeelint": "^1.14.2",
     "coveralls": "^2.11.6",
     "glob": "^6.0.4",
     "istanbul": "^0.4.2",
-    "mocha": "^2.4.4"
+    "mocha": "^2.4.4",
+    "sinon": "^1.17.3"
+  },
+  "dependencies": {
+    "faker": "github:marak/faker.js#9bb2b7c"
```

Comment:
Right. I went with the latest in master because it fixes a few issues like using the seeded prime generator instead of `Math.random()` for the UUID, which means in the latest release you would always get random values while in master you get consistent results with the same input seed. I'll check again in a few weeks to see if a new release was made. I've made #8 to keep track of this.